cmd/imagec/imagec.go

```diff
@@ -226,7 +230,7 @@ func DestinationDirectory() string {
 }
 
 // ImagesToDownload creates a slice of ImageWithMeta for the images that needs to be downloaded
-func ImagesToDownload(manifest *Manifest, storeName string) ([]*ImageWithMeta, error) {
+func ImagesToDownload(manifest *Manifest, storeName string) ([]*ImageWithMeta, *ImageWithMeta, error) {
```

Comment:
If all images (layers) already exist then images will be an empty slice. So the need for the additional ImageWithMeta -- in some cases it will be duplicated, but in others it won't.